examples/generate-test.md

```diff
@@ -1,10 +1,20 @@
-Use Playwright tools to generate test for scenario:
+## Instructions
```

Comment:
You should only include instructions here if you intend to remove the generate_test tool.